GLMakie/src/screen.jl

```diff
@@ -11,29 +11,29 @@ function renderloop end
 * `renderloop = GLMakie.renderloop`: sets a function `renderloop(::GLMakie.Screen)` which starts a renderloop for the screen.
 
 
-    !!! warning
-    The below are not effective if renderloop isn't set to `GLMakie.renderloop`, unless implemented in custom renderloop:
+!!! warning
+    The keyword arguments below are not effective if `renderloop` isn't set to `GLMakie.renderloop`, unless implemented in a custom renderloop function:
 
-
-* `pause_renderloop = false`: creates a screen with paused renderlooop. Can be started with `GLMakie.start_renderloop!(screen)` or paused again with `GLMakie.pause_renderloop!(screen)`.
-* `vsync = false`: enables vsync for the window.
-* `render_on_demand = true`: renders the scene only if something has changed in it.
-* `framerate = 30.0`: sets the currently rendered frames per second.
+* `pause_renderloop = false`: If `true`, creates a screen with a paused renderloop. The renderloop can be started with `GLMakie.start_renderloop!(screen)` and paused again with `GLMakie.pause_renderloop!(screen)`.
+* `vsync = false`: Whether to enable vsync for the window.
+* `render_on_demand = true`: If `true`, the scene will only be rendered if something has changed in it.
+* `framerate = 30.0`: Sets the currently rendered frames per second.
 
 ## GLFW window attributes
-* `float = false`: Lets the opened window float above anything else.
-* `focus_on_show = false`: Focusses the window when newly opened.
-* `decorated = true`: shows the window decorations or not.
+* `float = false`: Whether the window should float above other windows.
+* `focus_on_show = false`: If `true`, focuses the window when newly opened.
+* `decorated = true`: Whether or not to show window decorations.
 * `title::String = "Makie"`: Sets the window title.
-* `fullscreen = false`: Starts the window in fullscreen.
-* `debugging = false`: Starts the GLFW.Window/OpenGL context with debug output.
-* `monitor::Union{Nothing, GLFW.Monitor} = nothing`: Sets the monitor on which the Window should be opened.
+* `fullscreen = false`: Whether to start the window in fullscreen mode.
+* `debugging = false`: If `true`, starts the GLFW.Window/OpenGL context with debug output.
+* `monitor::Union{Nothing, GLFW.Monitor} = nothing`: Sets the monitor on which the window should be opened. If set to `nothing`, GLFW will decide which monitor to use.
```

Comment:
Would be nice to add how you actually set the monitor here...I think there's some discussion of this in #2544.

Reply:
Yeah, it looks like the current setting method doesn't do what you'd think it does (set the monitor on which the window should be displayed).

Since GLFW uses X directly, I guess the best way to do this would be to set `ENV["DISPLAY"]` before launching the window?!

Reply:
No, I don't think that's a good idea. I'm not 100% sure if that even works, and even if it does on linux, it won't work on windows.
So, I think we should just fix it to allow passing a `GLFW.Monitor`